docs/README.md

```diff
@@ -1,32 +1,31 @@
 ---
-
 home: true
 title: Waline
 heroImage: /logo.png
 actions:
+  - text: 💡 快速上手
+    link: /get-started.html
+    type: primary
 
-- text: 💡 快速上手
-  link: /get-started.html
-  type: primary
-
-- text: ℹ️ 介绍
-  link: /intro.html
+  - text: ℹ️ 介绍
+    link: /intro.html
+    type: secondary
 
 features:
+  - title: 自由评论
+    details: 完全的 Markdown 支持，同时包含表情、数学公式、HTML 嵌入
 
-- title: 自由评论
-  details: 完全的 Markdown 支持，同时包含表情、数学公式、HTML 嵌入
+  - title: 强大的安全性
+    details: 内容校验、防灌水、保护敏感数据等
 
-- title: 强大的安全性
-  details: 内容校验、防灌水、保护敏感数据等
+  - title: 登录支持
+    details: 在允许匿名评论的基础上，支持账号注册，保持身份
 
-- title: 登录支持
-  details: 在允许匿名评论的基础上，支持账号注册，保持身份
+  - title: 完全免费部署
+    details: 可免费部署在 Vercel 上
 
-- title: 完全免费部署
-  details: 可免费部署在 Vercel 上
-
-- title: 易于部署
-  details: 多种部署部署方式和存储服务支持
+  - title: 易于部署
+    details: 多种部署部署方式和存储服务支持
 
 footer: GPL-2.0 协议 | Copyright © 2020-present lizheming
+---
```

Comment:
BTW: This is called [frontmatter](https://jekyllrb.com/docs/frontmatter/) 

It should be surrounded by

```
---

---
```